publications.json

```diff
@@ -1,199 +1,307 @@
-            "slug": "slope",
-            "socialURLs": [
-                {
-                    "social": "insta",
-                    "URL": "https://www.instagram.com/slopemedia"
-                },
-                {
-                    "social": "facebook",
-                    "URL": "https://www.facebook.com/slopemedia"
-                },
-                {
-                    "social": "youtube",
-                    "URL": "https://www.youtube.com/user/SlopeMediaTV"
-                },
-                {
-                    "social": "spotify",
-                    "URL": "https://open.spotify.com/user/slopemedia?si=2_IbqlxtTP6O0fFkVp2GWA&nd=1"
-                }
-            ],
-            "websiteURL": "https://www.slopemedia.org"
-        },
-        {
-            "bio": "We created the review to provide a forum for the exchange of these ideas.",
-            "bioShort": "An open platform for scholarly writing, critical thinking, and reasoned debate",
-            "rssName": "The Undergraduate Law & Society Review at Cornell",
-            "rssURL": "https://www.culsr.org/?format=rss",
-            "name": "The Undergraduate Law and Society Review at Cornell",
-            "slug": "ulsr",
-            "socialURLs": [
-                {
-                    "social": "facebook",
-                    "URL": "https://www.facebook.com/CULSReview"
-                },
-                {
-                    "social": "linkedin",
-                    "URL": "https://www.linkedin.com/company/cornellundergraduatelawandsocietyreview/"
-                }
-            ],
-            "websiteURL": "https://www.culsr.org/"
-        }
-    ]
+   "publications":[
+      {
+         "bio":"The Advocate promotes the dissemination of literature that examines youth-related issues with the goal of encouraging youth advocacy in local and federal politics.",
+         "bioShort":"Encouraging political advocacy by examining issues relevant to college students",
+         "rssName":"The Advocate",
+         "rssURL":"http://theadvocatecornell.com/feed/",
+         "name":"The Advocate",
+         "slug":"advocate",
+         "socialURLs":[
+            {
+               "social":"insta",
+               "URL":"https://www.instagram.com/theadvocateatcornelluniversity"
+            },
+            {
+               "social":"facebook",
+               "URL":"https://www.facebook.com/The-Advocate-at-Cornell-University-109632540936869"
+            }
+         ],
+         "websiteURL":"http://theadvocatecornell.com"
+      },
+      {
+         "bio":"Student-run entertainment and media organization dedicated exclusively to Cornell University Athletics.",
+         "bioShort":"Dedicated exclusively to Cornell Athletics",
+         "rssName":"Big Red Sports Network",
+         "rssURL":"https://www.bigredsportsnetwork.org/feed",
+         "name":"Big Red Sports Network",
+         "slug":"brsn",
+         "socialURLs":[
```

Comment:
Can we flatten these and use the `social` field as the key and the `URL` as the value. We need to add these to the DB rather than make them field resolvers. Microphone should also add these fields to the DB, so that `volume-backend` can get them from the DB and not have a copy of this file